pkg/portForward/portForward.go

```diff
@@ -67,6 +67,10 @@ func (o *PFClient) StartPortForwarding(
 
 	o.StopPortForwarding()
 
+	if len(ceMapping) == 0 {
+		return nil
```

Comment:
From the end-user standpoint, could we display some message indicating that no ports will be forwarded? Or just `klog` the information?